server/controllers/admin/getDataForOneProduct.ts

```diff
@@ -0,0 +1,15 @@
+import { Request, Response } from 'express';
+import { getOneProduct } from '../../queries';
+import { CustomError } from '../../utils';
+
+const getOneProductForId = async (req:Request, res:Response) => {
+  const { productId } = req.params;
+  try {
+    const result = await getOneProduct(+(productId));
+    // console.log(result.map((e:any) => e.dataValues));
+    res.json(result);
+  } catch {
+    throw new CustomError(400, 'No Products');
+  }
+};
+export default getOneProductForId;
```

Comment:
Why is this controller?

Reply:
to get the one product data you tell me to do that